setup/configuration/index.md

```diff
@@ -23,7 +23,7 @@ The following table lists the Spinnaker services that currently incorporate exte
 
 ## Enabling external configuration
 
-To enable external configuration via the Config Server, add Spring Cloud Config configuration properties, which are under `spring.cloud.config.server`, and include settings for the backend you wish to use. If you deploy Spinnaker manually, you might place this configuration in a YAML file called `spinnakerconfig.yml`, alongside your `spinnaker.yml` file.
+To enable external configuration via the Config Server, add Spring Cloud Config configuration properties, which are under `spring.cloud.config.server`, and include settings for the backend you wish to use. If you are deploying Spinnaker using Halyard, you can place this file under the `~/.hal/$DEPLOYMENT/profiles` directory in the machine on which Halyard is installed, as described in [Custom Profiles](/reference/halyard/custom/#custom-profiles) (`$DEPLOYMENT` is typically `default`). If you deploy Spinnaker locally, you might place this configuration in a YAML file called `spinnakerconfig.yml`, alongside your `spinnaker.yml` file.
```

Comment:
```suggestion
To enable external configuration via the Config Server, add Spring Cloud Config configuration properties, which are under `spring.cloud.config.server`, and include settings for the backend you wish to use. 

If you are deploying Spinnaker using Halyard, you can place this file under the `~/.hal/$DEPLOYMENT/profiles` directory in the machine on which Halyard is installed, as described in [Custom Profiles](/reference/halyard/custom/#custom-profiles) (`$DEPLOYMENT` is typically `default`). If you deploy Spinnaker locally, you might place this configuration in a YAML file called `spinnakerconfig.yml`, alongside your `spinnaker.yml` file.
```

Reply:
@fifthposition , this presents Halyard-based deployment and local deployment as alternatives to each other, which I don't _think_ is right. Hal would be an alternative to manual editing of config files, and local deployment would be an alternative to distributed. Unless I'm misunderstanding what you're saying here.

Reply:
Moved back to "manually". Thanks Dave!